### PR TITLE
Add more system requirements and git ignore files

### PR DESCRIPTION
**Pull request by Claire-LHV**
- add more system requirements
- remove eclipse's specific files
- add eclipse's specific files to `gitignore`
- remove some description that the system cannot manage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,36 +2,39 @@
...
 - During the appointments, different health services like X-ray, blood work, special eye check, vaccinations, surgeries,... may be performed. All services have different code and price.
 - When a pet is registered, the pet's council registration number is optional. This will allow future changes of info and integration with the local council system for pets.
 - 1 veterinarian can have many bookings and many bookings can be done by 1 customer
-- There are two entities: specialist team and general practitioner team. Specialist team will perform complicated tasks such as surgery, while general practitioner team will perform simpler tasks like grooming, ..
+- There are two entities: specialist team and general practitioner team. Specialist team will perform complicated tasks such as surgery, while general practitioner team will perform simpler tasks like grooming, ...
+- Occasionally, PetCare gives out vouchers and discounts to pet rescuers
+- A user can make multiple login attempts. If correct credentials are given, login succeeds, otherwise, it fails.
```
Comment (minhn9753) [inline README.md:14]: This one should be in functional requirements. 
Minh 's idea: if an user fails to give correct credentials after 5 attempts, their accounts will be locked and they need to contact technical support for account recovery

File: README.md [hunk 2]
```diff
@@ -2,36 +2,39 @@
...
+- A user can change their password. Each password will only valid between the time when it's created and when it's changed to a new password.
+- An email will be sent to user's email address every time they change their password, everytime an appointment is made, everytime their details are changed, ...
 
 ## Non-functional requirements identified
 - Not too slow to load
 
 ## Prospective extended functionalities:
-- **(Prospect functionalities)** Integrates with vet on-premise staff (Receptionists, System Admins, Vets) system which has well-defined privileges; HR will have another system, which communicate with the staff system;...
+- **(security)** Encode users' passwords in the db.
+- **(functionalities)** Slot will only be reserved for 5 minutes since the time when the picking is selected.
```
Comment (minhn9753) [inline README.md:35]: 10 mins? 5 a bit short to me i think lol

File: README.md [hunk 3]
```diff
@@ -2,36 +2,39 @@
...
+- **(functionalities)** Integrates with a staff system which manages staff's privileges
+- **(functionalities)** Pet rescuers can register to be a registered pet rescuers to receive vouchers, discounts and donations.
 - **(security)** Supports users based in Australia; Blocks suspicious IP addresses; Firewall; Prevent DoS, DDoS;
-
-
+- **(functionalities)** Integrate with other third parties to extend payment options e.g, Afterpay, ...
+- **(functionalities)** For each successful login, the session will time out after a certain time of inactivity. User's account will be locked for 30 minutes and an email will be sent to user's email address after 6 failed login attempts.
```
Comment (minhn9753) [inline README.md:40]: Add one functionality (security): regarding password reset, an user cannot choose the last 3 passwords as a new password.

**Review by minhn9753 — COMMENTED** [review]
Apart from 3 comments, the rest looks good to me :)

File: README.md (hunk 1 above)
Comment (Claire-LHV) [reply README.md:14]: @minhn9753 
- I'll move it to functional requirements
- We have the max number of failed attempts in the extended functionalities. I think we should get it to work first and then enhance it later.

File: README.md (hunk 2 above)
Comment (Claire-LHV) [reply README.md:35]: Sounds good 👍  Going up to 10 minutes

File: README.md (hunk 3 above)
Comment (Claire-LHV) [reply README.md:40]: Good one 👍

**Review by kennguyen291 — COMMENTED** [review]
fixing the 5mins to 10mins as discussed and I think this is good to go.

**Review by kennguyen291 — APPROVED** [review]

**Pull request merged**